website/source/docs/http/sys-replication-primary.html.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: "http"
+page_title: "HTTP API: /sys/repliation/primary"
+sidebar_current: "docs-http-replication-primary"
+description: |-
+  The '/sys/replication/primary' endpoint focuses on managing replication operations for primary clusters. 
+---
+
+# /sys/replication/primary/enable
+
+## POST
+
+<dl>
+  <dt>Description</dt>
+  <dd>
+    Enables replication in primary mode. This is used when replication is currently disabled on the cluster
```

Comment:
I would add a sentence to the effect "Vault replication uses an primary-secondary model, where only one primary should be active at any given time. Multiple primaries may result in data loss."